docs/introduction/BeginnerTutorial.md

```diff
@@ -247,8 +250,8 @@ Well, `redux-saga` provides a way to make the above statement possible. Instead
 
 ```javascript
 // ...
+import { delay, takeEvery } from 'redux-saga'
```

Comment:
could you import `takeEvery` also from the `'redux-saga/effects'`? Its upcoming change and will be released before the weekend